### DISCUSSION TITLE
[16 gennaio 2024] Esercizio 3

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#223] [Esami]
- $\Large a)$ Convertire i numeri $\Large X=103$ e $\Large Y=68$ rappresentati in base 10 in complemento a 2 utilizzando 8 bits ed eseguire le operazioni $\Large Z=X-Y$ e $\Large W=X+Y$. Convertire i risultati in esadecimale.

- $\Large b)$ Considerare il numero con la virgola ottenuto usando gli 8 bit che rappresentano $\Large Y$ per la parte intera e gli 8 bit che rappresentano $\Large X$ per la parte decimale (considerare cioè $\Large Y,X$) e rappresentarlo nello standard IEEE 754 half precision.

- $\Large c)$ Eseguire l’operazione $\Large R=S+T$ tra i numeri $\Large S=1AB$ e $\Large T=2B7$ rappresentati in base 16.
    Convertire il risultato in base 10 e controllare la correttezza del risultato convertendo in base 10 gli operandi iniziali.
    
<details closed>
    <summary> soluzione A-L </summary>
<div align="center">

![image](https://github.com/user-attachments/assets/b6596d97-28fd-4e94-81ae-ced3c4b14cb1)
![image](https://github.com/user-attachments/assets/60c9c0d9-7d53-4787-9175-de91663379e7)

</div>
</details>

**Comment by M1SC0** [comment]
![image](https://github.com/user-attachments/assets/f7647e5c-a38f-4bc6-9c8d-5e1e5e4a61f9)
